build-system/pr-check/utils.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const colors = require('ansi-colors');
+const {execOrDie, exec, getStdout} = require('../exec');
+const {travisBuildNumber} = require('../travis');
+const BUILD_OUTPUT_FILE = `amp_build_${travisBuildNumber()}.zip`;
+const BUILD_OUTPUT_DIRS = 'build/ dist/ dist.3p/ EXTENSIONS_CSS_MAP';
+const BUILD_OUTPUT_STORAGE_LOCATION = 'gs://amp-travis-builds';
+
+/**
+ * Starts connection to Sauce Labs after getting account credentials
+ * @param {string} functionName
+ */
+function startSauceConnect(functionName) {
+  process.env['SAUCE_USERNAME'] = 'amphtml';
+  process.env['SAUCE_ACCESS_KEY'] = getStdout('curl --silent ' +
+      'https://amphtml-sauce-token-dealer.appspot.com/getJwtToken').trim();
+  const startScCmd = 'build-system/sauce_connect/start_sauce_connect.sh';
+  const fileLogPrefix = colors.bold(colors.yellow(`${functionName}:`));
+  console.log('\n' + fileLogPrefix,
+      'Starting Sauce Connect Proxy:', colors.cyan(startScCmd));
+  execOrDie(startScCmd);
+}
+
+/**
+ * Stops connection to Sauce Labs
+ * @param {string} functionName
+ */
+function stopSauceConnect(functionName) {
+  const stopScCmd = 'build-system/sauce_connect/stop_sauce_connect.sh';
+  const fileLogPrefix = colors.bold(colors.yellow(`${functionName}:`));
+  console.log('\n' + fileLogPrefix,
+      'Stopping Sauce Connect Proxy:', colors.cyan(stopScCmd));
+  execOrDie(stopScCmd);
+}
+
+/**
+ * Starts a timer to measure the execution time of the given function.
+ * @param {string} functionName
+ * @return {DOMHighResTimeStamp}
+ */
+function startTimer(functionName) {
+  const startTime = Date.now();
+  const fileLogPrefix = colors.bold(colors.yellow(`${functionName}:`));
```

Comment:
`fileLogPrefix` is currently the name of the file in which the execution. It used to be `pr-check.js`, but now it's the full gulp command, which looks goofy. See original implementation below:

https://github.com/ampproject/amphtml/blob/b84350059273cec5d8e683b3bfae073cf356dcd1/build-system/pr-check.js#L58-L63

For a quick fix, maybe take `fileLogPrefix` as an arg, and pass it in via a helper function that dynamically determines the name of the given file? (The prefix could be whatever you want it to be, it's merely meant to make Travis logs more readable.)